master/reference/node/requirements.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Calico System Requirements
+redirect_from: latest/reference/node/configuration
+---
+
+Depending on the Calico functionality  you are using, there are some requirements your system needs to meet in order for Calico to work properly.
+
+### For IPv4: minimum Linux kernel version: 2.6.32 
+### For IPv6: minimum Linux kernel version: 3.10
+
+## Requirements for Calico Policy:
+
+iproute2: iproute2 is a collection of utilities for controlling TCP/IP networking and traffic control in Linux.
+ Kernel modules dependencies: 
+- glibc
+- iptables
+- libelf
+ 
+iputils: The iputils package is set of small useful utilities for Linux networking.
+ Kernel modules dependencies: 
```

Comment:
I don't think this `|->` formats well in markdown - can we just make these bulleted lists?